doc/built-in-extensions.rst

```diff
@@ -66,3 +66,39 @@ necessary. This may change in the future.
           'javadoc-liberal': doccompat.javadoc_liberal,
           'kernel-doc': doccompat.kerneldoc,
       }
+
+Converting to Event Handling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have a function ``foo_transform()`` that you use with
+:py:data:`cautodoc_transformations`, it can be used with the
+:event:`hawkmoth-process-docstring` event as follows in ``conf.py``.
+
+Before:
+
+.. code-block:: python
+
+   cautodoc_transformations = {
+       'foo': foo_transform
+   }
+
+After:
+
+.. code-block:: python
+
+   def _process_docstring(app, lines, transform, options):
+       if transform != 'foo':
+           return
+
+       comment = '\n.join(lines)
+       comment = foo_transform(comment)
+       lines[:] = comment.splitlines()[:]
+
+   # conf.py can be turned into an extension by adding setup() function
```

Comment:
Not sure this is helpful. Shouldn't we steer the user to use `extensions = ['hawkmoth', '<extension>']` instead or is that not how it's supposed to work?

Reply:
Right, maybe it's more confusing than helpful. The idea was to describe how people can convert their existing solution to an extension using the event as simply as possible.

Reply:
Yeah, I got that, but it sounds kind of hackish and without context. I'd vote it out :)

Reply:
Dropped in #131.